libraries/resource_runit_service.rb

```diff
@@ -58,7 +58,7 @@ class RunitService < Chef::Resource::Service
       property :check_script_template_name, String, default: lazy { service_name }
       property :finish_script_template_name, String, default: lazy { service_name }
       property :control_template_names, Hash, default: lazy { set_control_template_names }
-      property :status_command, String, default: lazy { "#{sv_bin} status #{service_dir}" }
+      property :status_command, String, default: lazy { "#{sv_bin} status #{service_name}" }
```

Comment:
Does this even make to set at all here, since down below on line 107 we set this in after_created?
https://github.com/chef-cookbooks/runit/pull/245/files#diff-6772134783ea44796f91279ae48519daR107